File: web3rpc/rpc-specs/code-samples/python/debug/logging/setVMLogTarget.py
from web3 import Web3
from web3py_ext import extend

host = "https://api.baobab.klaytn.net:8651"

target = 3

w3 = Web3(Web3.HTTPProvider(host))
debug_response = w3.debug.set_vm_log_target(target)

print(debug_response)

File: web3rpc/rpc-specs/code-samples/javascript/debug/logging/setVMLogTarget.js
const { JsonRpcProvider } = require("@klaytn/ethers-ext");
(() => {
  const provider = new JsonRpcProvider("https://public-en-baobab.klaytn.net");
  const target = 3;

  provider.debug
    .setVMLogTarget(target, {}, (err, data, response) => {})
    .then((data) => {
      console.log(data);
    });
})();
